### DISCUSSION TITLE
Feature idea: Display definition/info instead of nothing

### DISCUSSION THREAD

**Discussion by mgkrupagmail** [sienori/simple-translate#437] [Ideas]
I absolutely love your add-on. I use it all the time and it works great. I just thought I'd share an idea that might make it even better.

Highlighting words in the target language (for me English) currently displays nothing (since I have the "Automatically switch to the second language" option unchecked).  But what if it displayed the definition instead. Basically making your "translate add-on" into a "translate or else define add-on". 

There are lots of "translate" add-ons and also lots of "define/dictionary" add-ons but in my searches, I've yet to encounter a "translate or else define" add-on so this feature might be a first and would make your add-on standout from the rest.

Perhaps the definition could be fetched from Wiktionary? or maybe its Wikipedia intro instead? Just an idea. 

Anyways, thanks again for the absolutely wonderful add-on.

**Comment by powerjungle** [comment]
Yeah but that wouldn't be "simple-translate" anymore. This needs to be a different addon in my opinion. I like the idea, but managing both at the same time seems unnecessary.
